pwnlib/util/misc.py

```diff
@@ -201,6 +202,14 @@ def run_in_new_terminal(command, terminal = None, args = None):
 
     Returns:
       PID of the new terminal process
+
+    Examples:
+        >>> type(run_in_new_terminal('/usr/bin/gdb -q /bin/ls'))
+        <type 'int'>
```

Comment:
This is an insufficient test.  You need to run something that generates output that can be checked.

Reply:
run_in_new_terminal opens a new terminal and writes to it, I am not sure how to check it from doctest, do you have any idea of how can I do that?

Reply:
Something that touches a temporary file would work.

```py
>>> pid = run_in_new_terminal(['touch', '/tmp/run-in-new-terminal-test'])
>>> pid > 0
True
>>> os.path.exists('/tmp/run-in-new-terminal-test')
True
```

Reply:
Updated my example to be a bit more robust